05_stringer/brvishnu92/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := `127.0.0.1
+`
+	actual := buf.String()
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func TestNumeronyms(t *testing.T) {
+	testCases := map[string]struct {
+		input    IPAddr
+		expected string
+	}{
+		"test1":  {input: IPAddr{127, 0, 0, 1}, expected: "127.0.0.1"},
+		"empty":  {input: IPAddr{}, expected: "0.0.0.0"},
+		"oneval": {input: IPAddr{127}, expected: "127.0.0.0"},
+	}
+	for name, test := range testCases {
+		test := test
+		t.Run(name, func(t *testing.T) {
+			result := test.input.String()
```

Comment:
The thing that is special about the Stringer interface is that it is called by functions in the `fmt` library. Your test should probably use a function from `fmt`

Reply:
Done.